packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -666,11 +668,11 @@ class TimelineEvent {
   @visibleForTesting
   TimelineEventType type;
 
-  /// Event start time in micros.
-  int startTime;
+  final TimeRange timeRange;
+
+  int get startTime => timeRange.start.inMicroseconds;
```

Comment:
I would remove these getters. The benefit I hope we get from the TimeRange abstraction is reducing the clutter of startTime and endTime getters everywhere in these classes.

Reply:
Removed.